src/common/scrub_types.cc

```diff
@@ -158,12 +158,22 @@ void inconsistent_snapset_wrapper::set_headless()
 
 void inconsistent_snapset_wrapper::set_ss_attr_missing()
```

Comment:
@tchaikov Even though this is a new feature I wanted to leave both the rados output "ss_attr_missing" and the user facing method ss_attr_missing() that returns bool the same.  So I want to leave set_ss_attr_missing() to correspond to them.  We could change all of it based on snapset_attr_missing.  Do you think we should do that?